apps/frontend/src/network/teams/__mocks__/api.ts

```diff
@@ -27,3 +27,5 @@ export const getTeams = jest.fn(
   async ({ pageSize }: GetListOptions): Promise<ListTeamResponse> =>
     createListTeamResponse(pageSize ?? 10),
 );
+
+export const createTeamResearchOutput = jest.fn();
```

Comment:
Could we please type this mock with the response output? I think we now return the draft RO right?